### DISCUSSION TITLE
Is it possible to "hide" an issue on github.com/issues without closing it?

### DISCUSSION THREAD

**Discussion by benblank** [community/community#29542] [Projects and Issues]
I'm making an effort to keep better track of the issues I've raised, for which [the Created tab on the Issues page](https://github.com/issues?q=is%3Aissue+is%3Aopen+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc) is invaluable.

However, some of the issues on that page are ones which I no longer expect to ever be addressed; for example, I raised an issue on a repository a little over ten years ago which has never seen a response. In fact, the repository seems to have been abandoned entirely around the same time.

No big deal; it happens.

However, that issue is now stuck at the bottom of my Created tab. I don't think that closing the issue is appropriate, as I feel that implies it has been brought to some kind of conclusion, which this has not. I see an `archived:false` term included in the filter, which looks promising, but I haven't been able to find any kind of documentation on "archiving" issues other than in the context of project boards.

Is there any way I can simply "hide" this issue from my default Created view?

**Comment by mazkdevf** [comment]
Hello benblank, Not basically but it is possible if you have for example private repository couldn't you just press 

![image](https://user-images.githubusercontent.com/79049205/184681823-b022f593-3da1-4634-a947-13b11aa05c3e.png)

Transfer Issue to another repository, then it will no longer show on the repository where its originally created.

**Reply by benblank** [reply]
Unfortunately, that isn't going to work for someone else's abandoned repo. 🙂

**Reply by mazkdevf** [reply]
> Unfortunately, that isn't going to work for someone else's abandoned repo. 🙂

Oh yeah, Sorry. 

I will hope that you will find an answer, but for now, Have a Good Day!

**Reply by benblank** [reply]
You, too! Thanks for contributing to the conversation.

**Reply by mazkdevf** [reply]
> 

👍🏻 Thanks

**Comment by hughlilly** [comment]
The `archived` flag appears to relate to whether the PR is [in a repo that is archived](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-based-on-whether-a-repository-is-archived).

I've never really liked the aggressiveness of the name “[wontfix](https://en.opensuse.org/Bug_Status_WONTFIX)”, but I’d say it would be great if anything tagged with that label were hidden by default. In other words, the default search could be "is:issue is:open -label:wontfix”.

Another option might be to [lock the conversation](https://docs.github.com/en/communities/moderating-comments-and-conversations/locking-conversations). I don’t think this would hide it by default though.
